.gitignore

```diff
@@ -2,7 +2,6 @@ node_modules
 dist
 bower-dist/*.map
 bower_components
-/.*
```

Comment:
Why was this removed?

Reply:
@hankwallace this ignores all files that start with a (dot), so we need a .npmrc file for getting the npm token.

Reply:
@nikhil-ht why not add a `!.npmrc` then like the lines below?

Reply:
@hankwallace  will add   !.npmrc